app/views/snps/_snpedia_papers.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="snpedia">
+  <h3>Links to SNPedia</h3>
+  <% if snp.snpedia_papers.present? %>
+    <table class="table table-striped">
+      <tr>
+        <th>Title</th>
+        <th>Summary</th>
+      </tr>
+      <% snp.snpedia_papers.each do |p| %>
+        <tr>
+          <td>
+            <%= bold_if_matching_local_genotype(p, user_snp) do %>
+              <%= link_to "#{snp.name} #{p.local_genotype.split('').join('/')}", p.url %>
+            <% end %>
+          </td>
+          <td><b><%= p.summary.capitalize %></b></td>
+        </tr>
+      <% end %>
+    </table>
+  <% else %>
+    There are no SNPedia-links yet. Please come back later.
+  <% end %>
+  <div class="small">
+    Note: For some SNPs, testing providers detect the genotype from the opposite strand of DNA, so that the genotypes listed at openSNP don't appear here. In those cases, please replace "A" by "T" or "G" by "C" (and vice-versa).
```

Comment:
What does this mean, btw.? Users should substitute A with T or G with C in their heads? Couldn't this be incorporated into the matching, so that links are emphasized that match, but were detected "from the opposite strand of DNA"?